docs/content.zh/docs/dev/datastream/execution_mode.md

```diff
@@ -170,7 +170,7 @@ TaskManagers 将至少在下游任务开始消费它们前保留中间结果（
 
 ### 事件时间/水印
 
-在支持[事件时间]({{< ref "docs/dev/datastream/event-time/generating_watermarks" >}})方面，Flink 的流运行时间建立在一个事件可能是乱序到来的悲观假设上的，即一个时间戳 `t` 的事件可能会在一个时间戳 `t+1` 的事件之后出现。因为如此，系统永远无法确定在给定的时间戳 `T` 下，未来不会再有时间戳 `t < T` 的元素出现。为了摊平这种失序性对最终结果的影响，同时使系统实用，在`流`模式下，Flink 使用了一种名为 [Watermarks]({{< ref "docs/concepts/time" >}}#event-time-and-watermarks) 的启发式方法。一个带有时间戳 `T` 的水印标志着再没有时间戳 `t < T` 的元素跟进。
+在支持[事件时间]({{< ref "docs/concepts/time" >}}#notions-of-time-event-time-and-processing-time)方面，Flink 的流运行时间建立在一个事件可能是乱序到来的悲观假设上的，即一个时间戳 `t` 的事件可能会在一个时间戳 `t+1` 的事件之后出现。因为如此，系统永远无法确定在给定的时间戳 `T` 下，未来不会再有时间戳 `t < T` 的元素出现。为了摊平这种失序性对最终结果的影响，同时使系统实用，在`流`模式下，Flink 使用了一种名为 [Watermarks]({{< ref "docs/concepts/time" >}}#event-time-and-watermarks) 的启发式方法。一个带有时间戳 `T` 的水印标志着再没有时间戳 `t < T` 的元素跟进。
```

Comment:
The English version of the documentation currently on master https://nightlies.apache.org/flink/flink-docs-master/docs/dev/datastream/execution_mode/#event-time--watermarks still points to the `docs/dev/datastream/event-time/generating_watermarks` page. I think the current situation is the right page to point users to and therefore we shouldn't make this change.